javascript-modal/main.js

```diff
@@ -0,0 +1,22 @@
+var click = true;
+var $openModal = document.querySelector('.openButton');
+var $textBoxContainer = document.querySelector('.textBoxContainer');
+var $overlay = document.querySelector('.overlay');
+var $noButton = document.querySelector('.noButton');
+
+$openModal.addEventListener('click', open);
+
+function open(event) {
+  if (click === true) {
```

Comment:
It doesn't look like this variable is ever changing in value. You can just delete the variable, and the conditionals referencing it.